pkg/common/common.go

```diff
@@ -59,16 +63,28 @@ func LogFileName() string {
 	return ConfigPath + "/" + RootCommandName + ".log"
 }
 
-func CliExit(msg interface{}) {
-	if msg != nil {
+func CliExit(err error) {
+	if err != nil {
 		_, file, line, _ := runtime.Caller(1)
-		log.WithFields(log.Fields{"file": file, "line": line}).Error(msg)
-		cobra.CheckErr(msg)
+		log.WithFields(log.Fields{"file": file, "line": line}).Error(err)
+
+		st, ok := status.FromError(err)
+
+		if ok {
+			switch (*st).Code() {
+			case codes.FailedPrecondition:
```

Comment:
Is this the only status we expect? Don't we need a default case?

Reply:
I'll add a default case, which is the same as the else clause here.